src/static/stilyagi/assets/js/roadmap.js

```
@@ -0,0 +1,63 @@
+      const phase = head.closest(".phase");
+      if (!phase) return;
+      phase.classList.toggle("open", open);
+      phase.classList.toggle("closed", !open);
+    };
+
+    const currentIndex = () =>
+      heads.findIndex((head) => head.getAttribute("aria-expanded") === "true");
+
+    const activate = (index) => {
+      const next = nextOpenIndex(currentIndex(), index);
+      heads.forEach((head, i) => setOpen(head, i === next));
```

Biome (2.5.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)